docs/quickstarts/chrome-extension.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Chrome Extension Quickstart
+description: Add authentication and user management to your Chrome Extension with Clerk.
+---
+
+# Chrome Extension Quickstart
+
+<TutorialHero
+  framework="nextjs"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Create a React application with Vite",
+      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project",
+      icon: "nextjs",
+    },
+    {
+      title: "Install the CRXJS Plugin",
+      link: "https://crxjs.dev/vite-plugin/getting-started/react/create-project",
+      icon: "nextjs",
+    }
+  ]}
+>
+
+- Install `@clerk/chrome-extension`
+- Set your environment variables
+- Add `<ClerkProvider>` to your application
+- Create a header with Clerk components for users to sign in and out
+- Add basic styles
+
+</TutorialHero>
+
+This tutorial only supports the email & password, email & email code, and SMS & SMS code [authentication options](/docs/authentication/configuration/sign-up-sign-in-options). Email & email link, social sign in (OAuth), SAML, and invites are not supported.
+
+<Steps>
+
+### Install `@clerk/chrome-extension`
+
+{/* TODO: Fill the link the reference */}
+Clerk's [Chrome Extension SDK](#) gives you access to prebuilt components, React hooks, and helpers to make user authentication easier.
```

Comment:
Is this a placeholder? Or is it meant to link to itself?

Reply:
yes there is a TODO right above it that says to fill the link once we create the reference in our docs.

Reply:
Yes, I totally missed that.

Reply:
Added link.